corehq/apps/importer/tasks.py

```diff
@@ -104,8 +104,11 @@ def _submit_caseblocks(caseblocks):
             if not any(fields_to_update.values()):
                 # if the row was blank, just skip it, no errors
                 continue
-        except importer_util.InvalidDateException:
-            errors.add(ImportErrors.InvalidDate, i + 1)
+        except importer_util.InvalidDateException as e:
+            errors.add(ImportErrors.InvalidDate, i + 1, e.column)
```

Comment:
To be honest, I'm not sure if that checkbox works in the way I expected it to:
When uploading a file without headers, I get the following... which seems to be a bug.
![screenshot from 2016-02-16 11-50-18](https://cloud.githubusercontent.com/assets/146896/13068644/9490137c-d4a3-11e5-805f-327378519030.png)

In answer to your question, I don't know.